HW1/P5/P5.txt

```diff
@@ -1 +1,160 @@
-# Your discussion here
+Note that helper functions are stored in the "network_commands" python module in the directory above this one.
+I wanted to place all of my network code in the same place.
+
+###### Shortest path #######
+
+The shortest path between 'Harvard_University' and 'Kevin_Bacon' is of length 3. One example is
```

Comment:
I don't understand what you mean, sorry. I guess it depends on how you define a path. If you define the path length as excluding the start and end node, sure. I excluded only the start node.